tmail-backend/event-bus-redis/src/test/java/org/apache/james/events/RabbitMQAndRedisEventBusTest.java

```diff
@@ -1,22 +1,3 @@
-/****************************************************************
- * Licensed to the Apache Software Foundation (ASF) under one   *
- * or more contributor license agreements.  See the NOTICE file *
- * distributed with this work for additional information        *
- * regarding copyright ownership.  The ASF licenses this file   *
- * to you under the Apache License, Version 2.0 (the            *
- * "License"); you may not use this file except in compliance   *
- * with the License.  You may obtain a copy of the License at   *
- *                                                              *
- *   http://www.apache.org/licenses/LICENSE-2.0                 *
- *                                                              *
- * Unless required by applicable law or agreed to in writing,   *
- * software distributed under the License is distributed on an  *
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
- * KIND, either express or implied.  See the License for the    *
- * specific language governing permissions and limitations      *
- * under the License.                                           *
- ****************************************************************/
-
```

Comment:
Retained attribution: 

```
/***************************************************************
* As a subpart of Twake Mail this file is edited by Linagora.                            *
*                                                                                                                                   *
* https://twake-mail.com/                                                                                      *
* https://linagora.com                                                                                            *
*                                                                                                                                  *
* This file is thus subject to The Affero Gnu Public License version 3.          *
*                                                                                                                                  *
* https://www.gnu.org/licenses/agpl-3.0.en.html                                             *
*                                                                                                                                  *
* This program is distributed in the hope that it will be useful,                     *
* but WITHOUT ANY WARRANTY; without even the implied warranty of     *
* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the     *
* GNU Affero General Public License for more details.                                    *
*                                                                                                                                  *
* This file was taken and adapted from the Apache James project                *
*                                                                                                                                  *
*    https://james.apache.org                                                                                 *
*                                                                                                                                  *
* It was originally licensed under the Apache V2 license.                                *
*                                                                                                                                  *
*    http://www.apache.org/licenses/LICENSE-2.0                                             *
*                                                                                                                                  *
 ***************************************************************/
```

We shall add in all of our files also the following header I believe:

```
/***************************************************************
* As a subpart of Twake Mail this file is edited by Linagora.                            *
*                                                                                                                                   *
* https://twake-mail.com/                                                                                      *
* https://linagora.com                                                                                            *
*                                                                                                                                  *
* This file is thus subject to The Affero Gnu Public License version 3.          *
*                                                                                                                                  *
* https://www.gnu.org/licenses/agpl-3.0.en.html                                             *
*                                                                                                                                  *
* This program is distributed in the hope that it will be useful,                     *
* but WITHOUT ANY WARRANTY; without even the implied warranty of     *
* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the     *
* GNU Affero General Public License for more details.                                    *
 ***************************************************************/
```